src/workerd/io/compatibility-date.h

```diff
@@ -48,4 +48,11 @@ kj::Array<kj::StringPtr> decompileCompatibilityFlagsForFl(CompatibilityFlags::Re
 kj::Maybe<kj::String> normalizeCompatDate(kj::StringPtr date);
 // Exposed to unit test the parser.
 
+static constexpr uint64_t COMPAT_ENABLE_FLAG_ANNOTATION_ID = 0xb6dabbc87cd1b03eull;
```

Comment:
For future generations, it's worth leaving a comment here mentioning where these `0x...` values come from.

Reply:
I can definitely add a comment! Where do the values come from?